core/src/main/scala/com/salesforce/op/OpWorkflowModel.scala

```diff
@@ -220,8 +223,19 @@ class OpWorkflowModel(val uid: String = UID[OpWorkflowModel], val trainingParams
    * @param path      path to save the model
    * @param overwrite should overwrite if the path exists
    */
-  def save(path: String, overwrite: Boolean = true): Unit = {
-    OpWorkflowModelWriter.save(this, path = path, overwrite = overwrite)
+  def save(path: String, overwrite: Boolean = true,
+    localDir: String = "tmp/model"): Unit = {
```

Comment:
1. please add docs for `localDir`
2. better use something random here, e.g. `localDir: String = s"tmp/model-${System.currentTimeInMillis}"`

Reply:
Consider calling it modelStagingDir. We don't need an explicit default, we can "borrow"/reflect into https://github.com/apache/spark/blob/065f17386d1851d732b4c1badf1ce2e14d0de338/core/src/main/scala/org/apache/spark/util/Utils.scala createTempDir and other goodies